experimental/IntersectionTheory/src/Main.jl

```diff
@@ -1768,14 +1793,14 @@ function abs_flag(dims::Vector{Int}; base::Ring=QQ, symbol::String="c")
 end
 
 @doc raw"""
-    abstract_flag_variety(F::AbstractBundle, d::Int; symbol::String="c")
-    abstract_flag_variety(F::AbstractBundle, dims::Vector{Int}; symbol::String="c")
+    abstract_flag_variety(F::AbstractBundle, dims::Int...; symbol::String = "c")
+    abstract_flag_variety(F::AbstractBundle, dims::Vector{Int}; symbol::String = "c")
 
 Given integers, say, $d_1, \dots, d_{k}$ or a vector of such integers, and given an
 abstract bundle $F$, return the abstract flag variety (flag bundle) of nested sequences 
 of subspaces of dimensions $d_1; \dots, d_{k}$ in the fibers of $F$.
 """
-function abstract_flag_variety(F::AbstractBundle, d::Int; symbol::String="c") abstract_flag_variety(F, [d], symbol=symbol) end
+function abstract_flag_variety(F::AbstractBundle, dims::Int...; symbol::String = "c") abstract_flag_variety(F, collect(dims), symbol=symbol) end
```

Comment:
for readability it would be better to have 
```suggestion
function abstract_flag_variety(F::AbstractBundle, dims::Int...; symbol::String = "c")
  abstract_flag_variety(F, collect(dims), symbol=symbol)
end

```
or
```suggestion
abstract_flag_variety(F::AbstractBundle, dims::Int...; symbol::String = "c") = abstract_flag_variety(F, collect(dims), symbol=symbol)

```